src/actions.ts

```diff
@@ -433,6 +433,11 @@ export function createActionsSystem(
     }
   }
 
+  function normalizeAnimationWeight(rawWeight: number): number {
+    if (rawWeight < 0 || rawWeight > 1) return 1
```

Comment:
if its less than 0, maybe the default should be 0 ? or the minimum is 1 ?

Reply:
Hey @gonpombo8, thanks for the review!

we are using the default value `1` here to keep consistency with the PBAnimationState component default value.

Less than `0` and higher than `1` are unexpected values, so we'll use the default value for both cases